vbfml/scripts/pickle_to_root.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import gzip
+import pickle
+import uproot
+import numpy as np
+
+from tqdm import tqdm
+from pprint import pprint
+from vbfml.util import vbfml_path
+
+pjoin = os.path.join
+
+
+def main():
```

Comment:
Add docstring explaining what this script is used for, what the input and outputs are

Reply:
Done in f8160d7